modules/virt-configuring-vm-disk-sharing.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-vm-disk-sharing{context}"]
+= Configuring disk sharing by using virtual machine disks
+
+You can configure a disk's block volumes to allow multiple virtual machines to share the same storage. 
+
+The application running in the guest determines the storage option you must configure for the VM. A disk of type `disk` exposes the volume as an ordinary disk to the VM.
+
+.Prerequisites 
+
+* `ReadWriteMany` (RWX) volume access mode is required if the VMs that are sharing disks are running on different nodes. If the VMs that are sharing disks are running on the same node, `ReadWriteOnce` (RWO) volume access mode is sufficient.
+
```

Comment:
Remove extra line break. Try to use the same construction for prerequisites. Also, try to use active, not passive, voice. Example: "The volume access mode must be RWX if..." 

Second sentence is OK since it is not a prerequisite but an explanation. You might want to break it to another line, joined with '+'.

Reply:
Done

Reply:
I am talking about double line breaks and the line break in line 24 (between first step and "+").

Reply:
I think I got them all now :)